src/api/app/components/fuzzy_time_component.rb

```diff
@@ -9,8 +9,14 @@ def initialize(time:)
   end
 
   def human_time_ago
-    return 'now' if (Time.now.utc - time) < 60
+    now = Time.now.utc
+    diff = now - time
+    return 'now' if diff.abs < 60
 
-    "#{time_ago_in_words(time)} ago"
+    if diff.positive?
+      "#{time_ago_in_words(time)} ago"
+    else
+      "in #{distance_of_time_in_words(now, time)}"
+    end
```

Comment:
I think you are correct actually. The only difference is a matter of what the method returns.
After trying it out with different options I'd say we can make a single use of `distance_of_time_in_words` for everything: it does not have the `now` option anymore, it will be `less than a minute` but I think it is even better. I'll align the method accordingly. :+1: